addons/mass_mailing/static/src/js/mailing_contact_tree.js

```diff
@@ -0,0 +1,34 @@
+/** @odoo-module **/
```

Comment:
``mailing_contact_view_list.js`` maybe ? As anyway code now calls it "list view", not "tree" anymore.